lib/src/main/java/com/diffplug/spotless/FormatterStep.java

```diff
@@ -28,7 +28,7 @@
  * to not introduce any windows-style newlines as well.
  */
 public interface FormatterStep extends Serializable, AutoCloseable {
-	/** The name of the step, for debugging purposes. */
+    /** The name of the step, for debugging purposes and to uniquely identify a step when computing the Gradle Build Cache key. */
```

Comment:
What happens if the implementation of the FormatterStep changes? I worry that the name alone is not enough, and that this could introduce negative cache hits.

Reply:
That's a very good point, I did not consider implementation changes

Reply:
Double thinking it, an implementation change should be shipped in a new version of the plugin, which would be reflected in the task classpath.